test/topology_experimental_raft/test_tablets.py

```diff
@@ -1049,6 +1050,24 @@ async def get_tablet_count_per_shard_for_host(shards_count: int, manager: Manage
                         result[shard_id] += 1;
     return result
 
+async def get_tablet_count_per_shard_for_hosts(manager: ManagerClient, servers: list[ServerInfo], full_tables: dict[str, list[str]], shards_per_node: int = 2) -> dict[ServerNum, list[int]]:
```

Comment:
Why this function cannot simply call `get_tablet_count_per_shard_for_host` for each of the hosts? Also I would appreciate if the order of arguments was the same, with only server being added as the last

Reply:
Because its complexity would be O(n^2) in the number of hosts.
If at all, I'd do it the other way around, where `get_tablet_count_per_shard_for_host` would call `get_tablet_count_per_shard_for_hosts` with a single server and manipulate the result to be backward compatible with the existing signature (for a single host)

Reply:
As for the parameters order, I moved `shards_per_node` to be last so it can be default-initialized.
Since the number of shards per host is not varied by the additional tests

Reply:
get_tablet_count_per_shard_for_host calls get_tablet_count_per_shard_for_hosts in next version

Reply:
> get_tablet_count_per_shard_for_host calls get_tablet_count_per_shard_for_hosts in next version

I would still like for them to have the same order of arguments as they as virtually the same, but we can leave it out for now and potentially change later. The fact that `get_tablet_count_per_shard_for_host` is just a "shortcut" is great improvement

Reply:
I reordered the parameters of `get_tablet_count_per_shard_for_host` in [268fcb5](https://github.com/scylladb/scylladb/commit/268fcb5f7cbd72782117dfc238552e9c0f509ba5):
```
async def get_tablet_count_per_shard_for_host(manager: ManagerClient, server: ServerInfo, full_tables: dict[str, list[str]], shards_count: int = 2) -> list[int]:
...
async def get_tablet_count_per_shard_for_hosts(manager: ManagerClient, servers: Iterable[ServerInfo], full_tables: dict[str, list[str]], shards_per_node: int = 2) -> dict[ServerNum, list[int]]:
```